hey.go

```diff
@@ -135,6 +135,10 @@ func main() {
 	// set content-type
 	header := make(http.Header)
 	header.Set("Content-Type", *contentType)
+
+	// set a default user-agent
+	header.Set("User-Agent", "hey (https://github.com/rakyll/hey)")
```

Comment:
Do we need a default user agent for hey? If so, we shouldn't override the existing user agent.